buildscripts/packaging/MacOS/make_dmg.sh

```
@@ -112,15 +112,13 @@ function change_rpath() {
 rm ${WORKING_DIRECTORY}/${COMPRESSEDDMGNAME}
 
 #tip: increase the size if error on copy or macdeployqt
-hdiutil create -size 750m -fs HFS+ -volname ${VOLNAME} ${WORKING_DIRECTORY}/${DMGNAME}
+hdiutil create -size 800m -fs APFS -volname ${VOLNAME} ${WORKING_DIRECTORY}/${DMGNAME}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)